content/ideas/algebraic-graphs.md

```diff
@@ -0,0 +1,62 @@
+---
+title: More graph algorithms for Alga
+---
+
+While graphs are ubiquitous in computer programming,
+they have long been underrepresented in Haskell
+due to the imperative nature and lack of type-safety
+of the existing implementations.
```

Comment:
This is a bit harsh towards the Functional Graph Library `fgl`, which provides a purely functional approach to modelling and manipulating graphs. I would say that the design space is not yet fully explored, and new approaches are being suggested by the community, such as algebraic graphs, which is the focus of this project.